public/coding/JavaScript/JsFunctions/greetings/greetings-Practice.js

```diff
@@ -1,5 +1,8 @@
+//Here's a little coding challenge. You're going to be writing a function that will take in 2 string parameters and output the desired sentence following this template
```

Comment:
if you change it here, you also have to reupload the zip file

Reply:
I completely forgot about that, thank you!